guides/micronaut-cloud-database-google/micronaut-cloud-database-google.adoc

```diff
@@ -0,0 +1,149 @@
+external:micronaut-cloud-database-base/start.adoc[]
+* You need a https://cloud.google.com/gcp/[Google Cloud Platform (GCP)] account and a GCP project.
+
+common:costs-gcloud.adoc[]
+
+common:gcp.adoc[]
+
+== Creating a MySQL Database instance in the Google Cloud
+
+We'll do the work of creating the database with the Google Cloud CLI. See the https://cloud.google.com/sdk/gcloud/reference/sql[MySQL CLI command reference] for more information.
+
+=== Create the MySQL instance
+
+[source,bash]
+----
+gcloud sql instances create micronaut-guides-mysql \
+ --database-version=MYSQL_8_0 \
+ --tier=db-f1-micro \
+ --region=us-east1 \
+ --root-password=<YOUR_ROOT_PASSWORD> \
+ --authorized-networks=<LOCAL_IP_ADDRESS>
+----
+* *_micronaut-guides-mysql_* - Change if you would like to use a different instance name
+* `--database-version` - Specifying MySQL v8.0 (See https://cloud.google.com/sql/docs/mysql/admin-api/rest/v1beta4/SqlDatabaseVersion[SQL DB Versions] for other options)
+* `--tier` - Change to match your needs. (Instead of using tiers, you can specify the CPU and memory, with the `--cpu` and `--memory` flags)
+* `--region` - Defaults to `us-central1` if omitted
+* `--root-password` - Default to `password123` if omitted
+* `--authorized-networks` - A single IP or a range of IP Addresses in the CIDR notation
+- `LOCAL_IP_ADDRESS` - This is the local IP address where the Micronaut application is running. You can find out the value your local machine by visiting http://whatismyip.akamai.com/[http://whatismyip.akamai.com/].
+
+[NOTE]
+====
+You might be prompted to enable the Google SQL Admin API
+
+[source, bash]
+----
+API [sqladmin.googleapis.com] not enabled on project [<project-id>].
+
+Would you like to enable and retry (this will take a few minutes)? (y/N)?
+----
+====
+
+You will see the following output:
+
+[source, bash, options="nowrap"]
+----
+NAME                    DATABASE_VERSION  LOCATION    TIER         PRIMARY_ADDRESS  PRIVATE_ADDRESS  STATUS
+micronaut-guides-mysql  MYSQL_8_0         us-east1-c  db-f1-micro  34.xxx.xxx.65    -                RUNNABLE
+----
+
+Make note of the _Primary Address_; you will need this later.
+
+If you need to update or change the `authorized-networks` use:
+[source, bash]
+----
+gcloud sql instances patch micronaut-guides-mysql \
+  --authorized-networks=<DIFFERENT_IP>
+----
+
+==== Create a User (Optional)
+
+Instead of using the _root_ DB User account, you can create one:
+
+[source, bash]
+----
+gcloud sql users create <USER_NAME> \
+  --instance=micronaut-guides-mysql \
+  --password=<USER_PASSWORD>
+----
+
+=== Create the Database
+
+Now that we have our MySQL instance running, we need to create a Database.
```

Comment:
```suggestion
Now that we have our MySQL instance running, we need to create a database.
```
"Database" is capitalized in some sentences and lowercase in others. I would recommend making sure they're all consistent, unless there's a specific reason that some of them are capitalized (like if they're class names).